### ISSUE TITLE
"Returned" dice relight under hothead rule

### ISSUE THREAD

**Issue by kortemaki** [TerryCavanagh/diceydungeons.com#2349]
Berlin Key "returns" the die used to trigger it.  In the case that this die is a six, not only does the die set to a six ignite, but the returned die also reignites, causing the player to double-pay for it.  This may be intentional, but the hothead rule text describes dice as igniting when "rolling" a six, not when "showing" a six.

If the behavior is intentional, you could rewrite the hothead rule as below:
> Your sixes always burn.

**Comment by TerryCavanagh** [comment]
Yep, ideally rerolled dice wouldn't burn when returned. I'll have a think about how to fix this, it's a little awkward implementation wise!
